src/Extensions/Logging/Internal/OmexLogger.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLogger : ILogger
+	{
+		public OmexLogger(OmexLogsEventSource logsEventSource, IExternalScopeProvider externalScopeProvider, string categoryName)
```

Comment:
Comments, applies throughout

Reply:
This class is internal and won't generate documentation